### PR TITLE
devel-project: reword notifier

### PR DESCRIPTION
**Pull request by lnussel**

### CODE REVIEW

**Comment by jberry-suse** [comment]
Tested package arg locally.

File: devel-project.py [hunk 1]
```diff
@@ -118,7 +120,22 @@ def notify(args):
     subject = 'Packages you maintain are present in {}'.format(args.project)
     for userid, package_identifiers in maintainer_map.items():
         email = entity_email(apiurl, userid)
-        message = 'The following packages you maintain are in {}:\n\n- {}'.format(
+        message = """This is a friedly reminder about your packages in {}.
```
Comment (jberry-suse) [inline devel-project.py]: `s/friedly/friendly/`

File: devel-project.py [hunk 2]
```diff
@@ -118,7 +120,22 @@ def notify(args):
...
+
+Please verify that the included packages are working as intended and
+have versions appropriate for a stable release. Please submit any
+changes until April 26th latest.
```
Comment (jberry-suse) [inline devel-project.py]: > Please submit any changes until April 26th latest.

This is a bit odd. Perhaps

```
Changes may be submitted until April 26th [at the latest].
```

**Comment by jberry-suse** [comment]
Since emails already sent the changes only matter for the future I suppose. :)

**Comment by jberry-suse** [comment]
Also note the idea was the tool could be used for SLE or Factory, but without adding templates they'd just need to adjust message.

**Comment by lnussel** [comment]
travis fail :(

**Comment by jberry-suse** [comment]
#1575

**Pull request merged**